src/github.com/getlantern/lantern/feed.go

```diff
@@ -199,10 +184,27 @@ func processFeed(provider FeedProvider) {
 		if desc == "" {
 			desc = entry.Content
 		}
-
 		feed.Entries[i].Description = desc
 	}
 
+	// the 'all' tab contains every article that's not associated with an
+	// excluded feed.
+	all := make(FeedItems, 0, len(feed.Entries))
+	for _, entry := range feed.Entries {
+		if !feed.Feeds[entry.Source].ExcludeFromAll {
+			all = append(all, entry)
+		}
+	}
+	feed.Items[allStr] = all
+
+	// Get a list of feed sources & send those back to the UI
+	for _, s := range feed.Sorted {
+		if entry, ok := feed.Feeds[s]; ok {
+			log.Debugf("Adding feed source: %s", entry.Title)
+			provider.AddSource(entry.Title)
+		}
```

Comment:
@myleshorton Logging both of those errors, and I'm checking for a missing title now